script/upgrades/MUGOV/MUGOVChecks.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.5.13;
+
+import { PrecompileHandler } from "celo-foundry/PrecompileHandler.sol";
+import { Test } from "forge-std/Test.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { console2 as console } from "forge-std/Script.sol";
+
+import { Chain } from "script/utils/Chain.sol";
+import { GovernanceScript } from "script/utils/Script.sol";
+import { IGovernanceFactory } from "script/interfaces/IGovernanceFactory.sol";
+import { IMentoToken, IEmission, IAirgrab, ITimelock, IMentoGovernor, ILocking } from "./interfaces.sol";
+
+contract MUGOVChecks is GovernanceScript, Test {
+  constructor() public {
+    new PrecompileHandler();
+    setUp();
+  }
+
+  bytes32 public airgrabMerkleRoot;
+  IGovernanceFactory public governanceFactory;
+
+  address mentoLabsMultisig;
+  address watchdogMultisig;
+  address fractalSigner;
+
+  IMentoToken mentoToken;
+  IEmission emission;
+  IAirgrab airgrab;
+  ITimelock governanceTimelock;
+  IMentoGovernor mentoGovernor;
+  ILocking locking;
+
+  function setUp() public {
+    contracts.load("MUGOV-04-Create-Factory", "latest");
+    airgrabMerkleRoot = readAirgrabMerkleRoot();
+    governanceFactory = IGovernanceFactory(contracts.deployed("GovernanceFactory"));
+
+    mentoToken = IMentoToken(governanceFactory.mentoToken());
+    emission = IEmission(governanceFactory.emission());
+    airgrab = IAirgrab(governanceFactory.airgrab());
+    governanceTimelock = ITimelock(governanceFactory.governanceTimelock());
+    mentoGovernor = IMentoGovernor(governanceFactory.mentoGovernor());
+    locking = ILocking(governanceFactory.locking());
+
+    mentoLabsMultisig = contracts.dependency("MentoLabsMultisig");
+    watchdogMultisig = contracts.dependency("WatchdogMultisig");
+    fractalSigner = contracts.dependency("FractalSigner");
+  }
+
+  function run() public {
+    // Balances:
+    assertEq(mentoToken.balanceOf(mentoLabsMultisig), 100000000 * 1e18, "MentoLabs multisig balance is incorrect");
+    assertEq(mentoToken.balanceOf(address(airgrab)), 100000000 * 1e18, "Airgrab balance is incorrect");
+    assertEq(
+      mentoToken.balanceOf(address(governanceTimelock)),
+      100000000 * 1e18,
+      "Governance timelock balance is incorrect"
+    );
+    assertEq(mentoToken.emissionSupply(), 693000000 * 1e18, "Emission supply is incorrect");
```

Comment:
some of these checks won't pass on mainnet.
we can come back to this laster if the allocations are not final